scripts/us_eia/open_data/generate_jsonl_for_bq.py

```diff
@@ -0,0 +1,103 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Converts a single dataset into 2 jsonl's: one each for series and category data.
+
+These jsonl's can then be imported to bq using the associated schema bq_schema_*.json
+
+To import to bigquery:
+- run this script: python3 generate_jsonl_for_bq.py
+- copy tmp_bq_import/ to gcs
+- bq load \
+    --source_format=NEWLINE_DELIMITED_JSON \
+    google.com:datcom-store-dev.import_us_eia.all_series \
+    gs://us_eia/bq_import/*.series.jsonl \
+    tmp_bq_import/bq_schema_series.json
+- bq load \
+    --source_format=NEWLINE_DELIMITED_JSON \
+    google.com:datcom-store-dev.import_us_eia.all_categories \
+    gs://us_eia/bq_import/*.categories.jsonl \
+    tmp_bq_import/bq_schema_categories.json
```

Comment:
nit: bq_schema_categories.json is not in tmp_bq_import?

Reply:
thanks for the catch! updated